frontend/sw360-portlet/src/main/webapp/html/utils/includes/searchUsers.jsp

```diff
@@ -23,24 +23,82 @@
 </portlet:resourceURL>
 
 <script>
+    firstRunUsers = true;
 
-    function searchUserAjax(what, how) {
-        return jQuery.ajax({
-            type: 'POST',
-            url: '<%=userSearchURL%>',
-            data: {
-                '<portlet:namespace/><%=PortalConstants.WHAT%>': what,
-                '<portlet:namespace/><%=PortalConstants.HOW%>': how
+    function showUserDialogWrapper(multiUser, resultInputId) {
+        require(['jquery', 'utils/includes/searchAndSelectIds'], function($, idSearch) {
+
+            $('#' + resultInputId + "Display").on('click', function() { showUserDialog(multiUser, resultInputId) });
+
+
+            function searchUserAjax(what, how) {
+                return jQuery.ajax({
+                    type: 'POST',
+                    url: '<%=userSearchURL%>',
+                    data: {
+                        '<portlet:namespace/><%=PortalConstants.WHAT%>': what,
+                        '<portlet:namespace/><%=PortalConstants.HOW%>': how
+                    }
+                });
+            }
+
+            function addAlreadySelectedUsersToTable(data, currentState) {
+                var tableData = "";
+
+                if (currentState.resultFullData.length > 0 && currentState.resultFullData[0] != "") {
+                    for (var i=0; i<currentState.resultFullData.length; i++) {
+                        var entry = currentState.resultFullData[i].split(",");
+                        tableData += '<tr>' +
+                            '<td><input type="' + ( currentState.multi ? 'checkbox' : 'radio' ) + '" checked="checked" value="' + entry.join(",") + '" name="id"/>' +'</td>' +
+                            '<td>' + entry[0] + '</td>' +
+                            '<td>' + entry[1] + '</td>' +
+                            '<td>' + entry[2] + '</td>' +
+                            '<td>' + entry[3] + '</td>' +
+                            '</tr>';
+                    }
+                }
+                tableData += data;
+                return tableData;
             }
-        });
-    }
 
-    var searchUser = new SearchAndSelectIds({
-        ajaxSearch: searchUserAjax
-    });
+            function renderUserInputToFullData(entry) {
+                return " , , " + entry + " , , " + entry;
+            }
+
+            const indexMail = 2;
+            const indexFullname = 4;
```

Comment:
What about this fullname? I mentioned earlier that it might not be needed anymore. If you can remove it, this code here has to be changed as well, I guess?!